plugins/engines/optix/CMakeLists.txt

```diff
@@ -28,8 +28,8 @@ set(BRAYNSOPTIXENGINEPLUGIN_PUBLIC_HEADERS
 )
 
 set(BRAYNSOPTIXENGINEPLUGIN_LINK_LIBRARIES
-    PUBLIC vmmlib braynsCommon braynsIO
-    PRIVATE braynsCommonEnginePlugin ${OptiX_LIBRARIES}
+  PUBLIC vmmlib braynsCommon braynsIO
+  PRIVATE braynsCommonEnginePlugin Lunchbox ${OptiX_LIBRARIES}
```

Comment:
Ideally, I would like the PTX code to be stored as a resource of the application. The CUDA_WRAP_SRCS macro allows the creation of OBJ files instead of PTX, but the OptiX API does not seem to make it possible to use this representation. Currently discussion with NVIDIA to see what is possible there.